src/rpdk/core/contract/suite/handler_commons.py

```diff
@@ -7,11 +7,29 @@
 
 def test_create_success(resource_client, current_resource_model):
     _status, response, _error_code = resource_client.call_and_assert(
-        Action.CREATE, OperationStatus.SUCCESS, current_resource_model
+        Action.CREATE,
+        OperationStatus.SUCCESS,
+        current_resource_model,
+        resource_client.primary_identifier_paths,
+    )
+    resource_client.assert_primary_identifier(
+        resource_client.primary_identifier_paths, current_resource_model
     )
     return response
 
 
+def test_create_fail(resource_client, resource_model):
```

Comment:
Please add comments or use a more explicit name to explain the test flow here, I don't really follow this one.

Reply:
update the method name to test_create_fail_if_read_only_property_in_input